layout/global-layer.html

```diff
@@ -26,22 +26,36 @@ <h2 id="when">When to Use</h2>
 <article>
 
   <h2 id="eyebrow">Eyebrow (aka Utility Nav)</h2>
-  <p>The eyebrow is more commonly referred to as the Pilot API Nav Header within the application frame because it provides for easy switching between product lines and platform level services. Platform level services are any services that exist across multiple product lines. Ex: Account Settings, Notifications, Ticketing, etc. </p>
+  <p>The eyebrow is more commonly referred to as the Pilot API Nav Header within the application frame because it provides for easy switching between product lines and platform level services. Platform level services are any services that exist across multiple product lines. Ex: Account Settings, Notifications, Ticketing, etc. For details specs please see the <a href="{{site.baseurl}}/layout/navigation#eyebrow">navigation page</a></p>
 
   <div class="row">
   	 <div class="col-md-4">
       <a class="anchor" name="top-nav"></a>
         <ul>
           <li>Extends across the top of the entire Helix-UI and is always visible</li>
-          <li>Contains links to platform level services and the <a href="#global">global dashboard</a></li>
+          <li>Contains links to platform level services, account management, product picker and the <a href="#global">global dashboard</a></li>
+          <li>Composed of 5 breakpoints that adapt to the users device and window size</a></li>
```

Comment:
Take the `</li>` at the end of this line, and place it on a new line after line 44. [Reference](https://www.w3.org/wiki/HTML_lists#Nesting_lists)